addons/progress-bar/addon.json

```
@@ -32,7 +32,7 @@
       "type": "integer",
       "min": 3,
       "max": 25,
-      "default": 5
+      "default": 10
```

```suggestion
      "default": 8
```
I think 10 pixels is a little too big in the editor menubar to be used as the default.